src/contexts/account/useWithdrawals.ts

```diff
@@ -100,6 +102,8 @@ const formatPendingWithdrawals = (values: string[], timestamps: string[]): Pendi
   return pendingWithdrawals.reverse();
 };
 
+const pendingWithdrawalsLoadInterval = 60 * 1000;
```

Comment:
Is this meant to be on a much faster interval than other actions?

Reply:
Yes. Showing up to date pending withdrawals (e.g. after withdrawal call to bot) is important from the user perspective, so it's better to do this more often.